packages/store-sdk/src/analytics/events.ts

```diff
@@ -0,0 +1,185 @@
+export interface PageViewData {
+  accountName: string
+  pageTitle: string
+  pageUrl: string
+  referrer: string
+  pageType: 'home' | 'fullText' | 'pdp' | 'plp'
+}
+
+export interface UserData extends PageViewData {
+  firstName?: string
+  lastName?: string
+  document?: string
+  id?: string
+  email?: string
+  phone?: string
+  isAuthenticated: boolean
+}
+
+export interface CartIdData extends PageViewData {
+  cartId: string
+}
+
+export type ProductPageInfoData = PageViewData
+
+export interface CategoryViewData extends PageViewData {
+  id: string
+  name: string
+}
+
+export interface DepartmentViewData extends PageViewData {
+  id: string
+  name: string
+}
+
+export interface InternalSiteSearchViewData extends PageViewData {
+  term: string
+  results: number
+}
+
+export interface AddToCartData {
+  products: CartPixelProduct[]
+  oneClickBuy?: boolean
```

Comment:
what's the purpose of this oneClickBuy?

Reply:
It was used to avoid navigating to the checkout before the addItems request was made. It'll be removed.